api/test/e2e/route_with_plugin_orchestration_test.go

```diff
@@ -0,0 +1,111 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package e2e
+
+import (
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestRoute_With_Plugin_Orchestration(t *testing.T) {
+	bytes, err := ioutil.ReadFile("../testdata/dag-conf.json")
+	assert.Nil(t, err)
+	dagConf := string(bytes)
+	bytes, err = ioutil.ReadFile("../testdata/invalid-dag-conf.json")
```

Comment:
Should we point out where it is invalid ? Or we can comment it.

Reply:
added a comment.